projects/plugins/jetpack/_inc/client/recommendations/prompts/resource-prompt/index.jsx

```diff
@@ -133,7 +133,7 @@ const ResourcePromptComponent = props => {
 					</ExternalLink>
 					<div className="jp-recommendations-question__jump-nav">
 						<a href={ nextRoute } onClick={ onResourceSkipClick }>
-							{ __( 'Read Later', 'jetpack' ) }
+							{ __( 'Not now', 'jetpack' ) }
```

Comment:
Changed this to be the same as the feature prompt because this agency recommendation isn't a "feature" to "activate", but it's also not an article to read. Thought it made sense to make this a bit more neutral/flexible in meaning